address-book.json

```diff
@@ -1,6 +1,9 @@
 {
   "1": {
     "Token": {
+      "address": "0x6B175474E89094C44Da98b954EedeAC495271d0F"
+    },
+    "SAIToken": {
       "address": "0x89d24A6b4CcB1B6fAA2625fE562bDD9a23260359",
       "bytecode": "0xa288eb5e119c76a1fc65ceb646ecf0a77657f66bb08038a226a37a467a43d035",
       "txHash": "0x91b318fd3498c14a1495357d6a4e895fd29c3d915ee695a6fbc8ef72908a1aa8"
```

Comment:
why do we have the address/bytecode/txhash for sai but we dont have it for dai (token) on mainnet?

Reply:
It was already there for `Token`, we are moving `Token` (i.e. SAI) to `SAIToken` so I just changed the name of the JSON key. There's no way we can rely on the bytecode and txHash since we aren't going to be the ones that deploy these contracts.